src/runner.rs

```diff
@@ -177,16 +177,14 @@ fn name_end_char(c: char) -> bool {
     c == ' ' || c == '='
 }
 
-/// Check if an argument string affects the Firefox profile
+/// Check if an argument string matches a given name
 ///
 /// Returns a boolean indicating whether a given string
-/// contains one of the `-P`, `-Profile` or `-ProfileManager`
-/// arguments, respecting the various platform-specific conventions.
-pub fn is_profile_arg(arg: &str) -> bool {
+/// contains an argument of a given names, respecting
+/// various platform-specific conventions.
```

Comment:
I don’t think “given string” is very good documentation. I suggest something like this, provided `names` is renamed `matches`:

> Check that a platform-specific argument string `arg` or arbitrary length (for example `/Profile C:\\some\\path` or `-p test -something -else`) contains one of the flags given in the array of `matches`, which should be given _without_ leading dashes, e.g. `&["p", "profile"]`.